modules/common-attributes.adoc

```diff
@@ -9,3 +9,4 @@
 :prewrap!:
 :op-system-first: Red Hat Enterprise Linux CoreOS (RHCOS)
 :op-system: RHCOS
+:asb-name: OpenShift Ansible broker
```

Comment:
asb-name is 'Ansible Service Broker', this will be more accurate.

Reply:
I remember we were using 'Openshift Ansible broker' as official name in last several OCP release docs. 
@shawn-hurley How do you think?